pkg/management/postgres/probes.go

```diff
@@ -415,8 +510,8 @@ func (instance *Instance) fillWalStatusFromConnection(result *postgres.Postgresq
 			coalesce(sync_state, ''),
 			coalesce(sync_priority, 0)
 		FROM pg_catalog.pg_stat_replication
-		WHERE application_name LIKE $1 AND usename = $2`,
-		fmt.Sprintf("%s-%%", instance.ClusterName),
+		WHERE application_name ~ $1 AND usename = $2`,
+		fmt.Sprintf("%s-[0-9]+$", instance.ClusterName),
```

Comment:
Personally I don't know if this changes should be in this patch

Reply:
the change to this SQL is to filter out rows that are not represented as replication to standby instances